src/widgets/DataTable/DataTableRow.js

```diff
@@ -29,6 +29,8 @@ import TextInputCell from './TextInputCell';
 
 import { formatStatus } from '../../utilities/index';
 
+import { COLUMN_TYPES } from '../../pages/dataTableUtilities/constants';
```

Comment:
forgot this one: export from `dataTableUtilities/index`

Reply:
4d1161b32797fb3e027fcae76df281c1063ceec1.